src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,43 +1,45 @@
 import * as React from "react"
 import { DefaultProps } from "../types"
 import styled from "../utils/styled"
+import { CaretLeftIcon } from "../Icon/Icon"
 
 export interface BreadcrumbsProps extends DefaultProps {
   /** Children as `Breadcrumb` elements */
   children?: React.ReactNode
 }
 
-const Container = styled("div")({
+const Container = styled("nav")(({ theme }) => ({
   label: "breadcrumbs",
+  color: theme.color.text.action,
   "& > *": {
-    display: "inline-block",
+    verticalAlign: "middle",
   },
-  "& a": {
-    textDecoration: "none",
-    color: "inherit",
-  },
-})
+}))
 
 const Slash = styled("span")(({ theme }) => ({
-  display: "inline-block",
-  margin: `0 ${theme.space.medium}px`,
+  margin: `0 ${theme.space.content}px`,
   color: theme.color.text.lightest,
-  ":first-child": {
-    marginLeft: 0,
-  },
 }))
 
+const addCaret = (item: React.ReactElement) => React.cloneElement(item, { icon: CaretLeftIcon, iconLeft: true })
+
 /**
  * Intersperse slashes between the children (`<Breadcrumb />` elements)
  * Curried first argument is necessary to give unique auto-incrementing
  *  keys to the slash elements.
  */
-
 const intersperseSlashes = (index: number) => ([head, ...tail]: React.ReactNode[]): React.ReactNode[] =>
-  head ? [<Slash key={`divider-${index}`}>/</Slash>, head, ...intersperseSlashes(index + 1)(tail)] : []
+  head && tail.length
+    ? [head, <Slash key={`divider-${index}`}>/</Slash>, ...intersperseSlashes(index + 1)(tail)]
+    : [head]
 
-const Breadcrumbs: React.SFC<BreadcrumbsProps> = props => (
-  <Container {...props}>{intersperseSlashes(0)(React.Children.toArray(props.children))}</Container>
-)
+const Breadcrumbs: React.SFC<BreadcrumbsProps> = props => {
+  const items = React.Children.toArray(props.children)
+  return (
+    <Container aria-label={"Breadcrumb"} {...props}>
```

Comment:
🤔  is this helpful label? If we want more semantics we can use `<nav>` element

Reply:
It's not helpful. I suggest `aria-label={children}` or similar if we have access to the actual text, or we can use a `nav`.

Reply:
I was so wrong. This actually implements the spec perfectly. My bad.

Reply:
We can perhaps do this though

```suggestion
    <Container aria-label="Breadcrumb" {...props}>
```